tests/smtp-md5/test.yaml

```diff
@@ -0,0 +1,27 @@
+requires:
+  features:
+    - HAVE_LIBJANSSON
+    - HAVE_NSS
+
+args:
+  - --set app-layer.protocols.smtp.mime.body-md5=yes
```

Comment:
is there a reason to set it here instead of in the yaml?

Reply:
Ah, no..  Was hoping to get by without adding any app-layer section at first.  But turns out our smtp tests don't actually pass with 0 config suggesting our default suricata.yaml doesn't match the smtp app-layer defaults.  But thats a problem for another day.  Will move it into the suricata.yaml.